src/graph/test/IndexTest.cpp

```diff
@@ -125,11 +125,13 @@ TEST_F(IndexTest, TagIndex) {
         std::string query = "SHOW TAG INDEX STATUS";
         auto code = client->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
-        std::vector<uniform_tuple_t<std::string, 2>> expected{
-            {"single_person_index", "SUCCEEDED"},
-            {"multi_person_index",  "SUCCEEDED"},
-        };
-        ASSERT_TRUE(verifyResult(resp, expected));
+        /*
+         * Currently , expected the index status is "RUNNING" or "SUCCEEDED"
+         */
+        for (auto& row : *resp.get_rows()) {
```

Comment:
I want to know the original purpose about this test case.

Reply:
> I want to know the original purpose about this test case.

Actually, I think this test case without any value at here(except for syntax checking). Because the data validity has been verified in other test cases. Therefore, the results of this asynchronous execution are best not checked here.
If it were me, I'd rather delete it.

Reply:
:( 
Fine.  Let's leave it here right now.